### DISCUSSION TITLE
CKEditor5 + Froala + Tinymce6 into GrapesJS

### DISCUSSION THREAD

**Discussion by GoodPHP** [GrapesJS/grapesjs#4671] [Show and tell]
### GrapesJS version

- [X] I confirm to use the latest version of GrapesJS

### What browser are you using?

Chrome

### Reproducible demo link

-

### Describe the bug

CKEditor5:
https://gjs.market/products/tinymce-6-inline-text-editor

Froala:
https://gjs.market/products/froala-inline-text-editor

Tinymce6:
https://gjs.market/products/tinymce-6-inline-text-editor

Sorry :)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by micker** [comment]
interesting feature

**Comment by GoodPHP** [comment]
> interesting feature

Thanks ✅

**Comment by mosh-tudor** [comment]
Btw, do you think it's ok/legal to do things like this:
```js
return e.editor ? e.editor.html.get().replace(/<p data-f-id="pbf" [^>]*>Powered by <a[^>]* title="Froala Editor">Froala Editor<\/a><\/p>/i, "") : ""
```

?
